openapi.yaml

```diff
@@ -14404,22 +14407,52 @@ paths:
-                    - $ref: '#/components/schemas/NodeBalancerConfig'
-                    - type: object
-                      properties:
-                        nodes:
-                          type: array
-                          description: >
-                            The NodeBalancer Nodes that serve this port.  When
-                            creating a NodeBalancer this way, at least one Node
-                            is required per configured port.
-                          items:
-                            $ref: '#/components/schemas/NodeBalancerNode'
+                    type: object
+                    description: A request object representing a NodeBalancer Config, including Nodes.
+                    properties:
+                      port:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/port'
+                      protocol:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/protocol'
+                      algorithm:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/algorithm'
+                      stickiness:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/stickiness'
+                      check:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check'
+                      check_interval:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check_interval'
+                      check_timeout:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check_timeout'
+                      check_attempts:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check_attempts'
+                      check_path:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check_path'
+                      check_body:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check_body'
+                      check_passive:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/check_passive'
+                      proxy_protocol:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/proxy_protocol'
+                      cipher_suite:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/cipher_suite'
+                      ssl_cert:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/ssl_cert'
+                      ssl_key:
+                        $ref: '#/components/schemas/NodeBalancerConfig/properties/ssl_key'
+                      nodes:
+                        type: array
+                        description: |
+                          The NodeBalancer Node(s) that serve this Config.
+                        items:
+                          $ref: '#/components/schemas/NodeBalancerNode'
```

Comment:
This is expanded because `items` defined with `allOf` currently don't render on the docs site. Can be reverted after a fix is made, which I've already requested.